src/adera/Machines/Rocket.cpp

```diff
@@ -182,10 +236,22 @@ Machine& SysMachineRocket::instantiate(ActiveEnt ent, PrototypeMachine config,
     BlueprintMachine settings)
 {
     // Read engine config
-    float thrust = std::get<double>(config.m_config["thrust"]);
-    
+    MachineRocket::Parameters params;
+    params.m_maxThrust = std::get<double>(config.m_config["thrust"]);
+    params.m_specImpulse = std::get<double>(config.m_config["Isp"]);
+
+    MachineRocket::fuel_list_t inputs;
+    Package& pkg = m_scene.get_application().debug_find_package("lzdb");
+    DependRes<ShipResourceType> fuel = pkg.get<ShipResourceType>("fuel");
+    if (!fuel.empty())
+    {
+        ActiveEnt fuelSource = static_cast<ActiveEnt>(13);
```

Comment:
So this is pretty egregious, but generalizing this stuff is going to take this PR from +500 lines to +700, so perhaps this is acceptable for the time being

Reply:
I'm just manually assigning the feeding tank to the rocket.

Yes, it should be searching, but since any tank anywhere on the ship can be attached to any rocket anywhere on the ship, it needs a bit of an involved solution. I guess I could give it a config value for the time being and read that, it would just push the hard-codedness to the part config files. I will fix this in the next PR or two, but if you want to change it for now let me know how you think we should handle it. I can include my subsequent commits in this branch too, it'll just make the PR twice as bloated.